blueprints/ember-bootstrap-modals-manager/index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env node */
+
+const rsvp = require('rsvp');
+
+module.exports = {
+  name: 'ember-bootstrap-modals-manager',
+  description: 'Configure ember-bootstrap-modals-manager',
+
+  afterInstall() {
+    return this.addEmberBootstrap();
+  },
+
+  addEmberBootstrap() {
+    const dependencies = this.project.dependencies();
+    const promises = [];
+    if (!('ember-bootstrap' in dependencies)) {
+      promises.push(this.addAddonToProject('ember-bootstrap'));
```

Comment:
I had this `rsvp.all` and `promises.push()` stuff around because I was installing more that one thing, which all return a promise. But in this case it's enough to just `return this.addAddonToProject('ember-bootstrap');`. Also no need for `rsvp` as a dependency then! Otherwise looks good! 👍